lambda/src/main/scala/pricemigrationengine/migrations/HomeDelivery2025Migration.scala

```diff
@@ -97,6 +98,16 @@ object HomeDelivery2025Migration {
     }
   }
 
+  def decideShouldRemoveDiscount(item: CohortItem): Boolean = {
+    val flag_opt = (for {
+      attributes <- item.migrationExtraAttributes
+      data: Newspaper2025ExtraAttributes =
+        upickle.default.read[Newspaper2025ExtraAttributes](attributes)
```

Comment:
```suggestion
      data: HomeDelivery2025ExtraAttributes =
        upickle.default.read[HomeDeliveryExtraAttributes](attributes)
```

to avoid confusion between HomeDelivery2025 and Newspaper2025 migrations?